base/strings/util.jl

```diff
@@ -400,6 +400,7 @@ single character, a vector or a set of characters, a string, or a regular expres
 is a function, each occurrence is replaced with `r(s)` where `s` is the matched substring.
 If `pat` is a regular expression and `r` is a `SubstitutionString`, then capture group
 references in `r` are replaced with the corresponding matched text.
+To delete patterns in `pat` set `r` to an empty String (`""` or `string()`).
```

Comment:
I'm okay with only mentioning `""` but `''` does not work. I was trying to use the function with an empty Char at first, but it seems like its not possible to create an empty Char.
```julia
julia> ''
ERROR: syntax: invalid character literal
```